server/views/referrals/courseParticipations/index.njk

```diff
@@ -5,6 +5,28 @@
 
 {% extends "../../partials/layout.njk" %}
 
+{% macro actionForm(secondaryActionText = "Add a programme") %}
```

Comment:
Does the argument need a default value? Might be clearer in the code below if it's passed explicitly in both contexts?